lib/src/crop.dart

```diff
@@ -70,6 +71,14 @@ class _CropState extends State<Crop> with TickerProviderStateMixin {
   Offset _previousOffset = Offset.zero;
   Offset _startOffset = Offset.zero;
   Offset _endOffset = Offset.zero;
+  double _previousRotation = 0.0;
```

Comment:
This variable is defined but never used.

Reply:
HR updated.